ui/webui/src/components/storage/InstallationMethod.jsx

```diff
@@ -411,25 +431,136 @@ const InstallationDestination = ({ deviceData, diskSelection, dispatch, idPrefix
+    console.log("Spawning Blivet.");
+    cockpit.spawn(["blivet-gui"]).then(() => console.log("Blivet exited."));
+    // We don't have an event informing that Blivet started so just wait a bit.
+    window.setTimeout(onStarted, 1000);
+    onStart();
+};
+
+const StorageModifiedModal = ({ onClose, onRescan }) => {
+    return (
+        <Modal
+          id="storage-modified-modal"
+          title={_("Modified storage")}
+          isOpen
+          variant="small"
+          showClose={false}
+          footer={
+              <>
+                  <Button
+                    onClick={() => { onClose(); onRescan() }}
+                    variant="primary"
+                    id="storage-modified-modal-rescan-btn"
+                    key="rescan"
+                  >
+                      {_("Rescan storage")}
+                  </Button>
+                  <Button
+                    variant="secondary"
+                    onClick={() => onClose()}
+                    id="storage-modified-modal-ignore-btn"
+                    key="ignore"
+                  >
+                      {_("Ignore")}
+                  </Button>
+              </>
+          }>
+            {_("If you have made changes on partitions or disks, please rescan storage.")}
+        </Modal>
+    );
+};
+
+const ModifyStorageModal = ({ onClose, onToolStarted }) => {
+    const [toolIsStarting, setToolIsStarting] = useState(false);
+    const onStart = () => setToolIsStarting(true);
+    const onStarted = () => { setToolIsStarting(false); onToolStarted() };
+    return (
+        <Modal
+          id="modify-storage-modal"
+          title={_("Modify storage")}
+          isOpen
+          variant="small"
+          titleIconVariant="warning"
+          showClose={false}
+          footer={
+              <>
+                  <Button
+                    onClick={() => startBlivet(
+                        onStart,
+                        onStarted
+                    )}
+                    id="modify-storage-modal-modify-btn"
+                    icon={toolIsStarting ? null : <ExternalLinkAltIcon />}
+                    isLoading={toolIsStarting}
+                    isDisabled={toolIsStarting}
+                    variant="primary"
+                  >
+                      {_("Launch Blivet storage editor")}
+                  </Button>
+                  <Button
+                    variant="link"
+                    onClick={() => onClose()}
+                    id="modify-storage-modal-cancel-btn"
+                    key="cancel"
+                    isDisabled={toolIsStarting}
+                  >
+                      {_("Cancel")}
+                  </Button>
+              </>
+          }>
+            <TextContent>
+                <Text component={TextVariants.p}>
+                    {_("Blivet is and advanced storage editor that lets you resize, delete, and create partitions. It can set up LVM and much more.")}
```

Comment:
I guess this should be "Blivet GUI" instead of just "Blivet" ? I'm thinking that saying "Blivet GUI" also makes it possible users might find relevant documentation/HOWTOs instead of possibly finding the Blivet library refferrence.

Reply:
Updated in a separate commit.